internal/pkg/template/templates/workloads/services/rd-web/cf.yml

```diff
@@ -134,6 +134,11 @@ Resources:
           EgressType: VPC
           VpcConnectorArn: !Ref VpcConnector
       {{- end }}
+      {{- if and (contains .FeatureFlags "Tracing") (eq .Tracing "AWSXRAY")}}
+      ObservabilityConfiguration:
+        ObservabilityEnabled: true
+        ObservabilityConfigurationArn: !Sub 'arn:aws:apprunner:${AWS::Region}:${AWS::AccountId}:observabilityconfiguration/DefaultConfiguration/1/00000000000000000000000000000001'
```

Comment:
Are they going to create a default configuration for each region with ARN like this?